examples/docker-compose-tailscale.yml

```
@@ -0,0 +1,36 @@
+version: "3"
+services:
+  tailscale-client:
+    image: tailscale/tailscale:latest
+    container_name: tailscale
+    hostname: tailscale-minecraft # This name will be the one on the tailscale network
+    environment:
+      TS_AUTHKEY: "tskey-auth-PLACE-YOUR-KEY-HERE"
+      TS_STATE_DIR: "/var/lib/tailscale"
+      TS_USERSPACE: "FALSE" # If not using Auto-pause it can be set as true
+    volumes:
+      - ${PWD}/ts-minecraft/state:/var/lib/tailscale

```

Being a compose file you can use relative host directory here

```suggestion
      - ./ts-minecraft/state:/var/lib/tailscale
```